mithril-client-cli/src/commands/cardano_db/download.rs

```diff
@@ -60,23 +60,51 @@ pub struct CardanoDbDownloadCommand {
 }
 
 impl CardanoDbDownloadCommand {
-    /// Is JSON output enabled
-    pub fn is_json_output_enabled(&self) -> bool {
-        self.shared_args.json
-    }
-
     /// Command execution
     pub async fn execute(&self, context: CommandContext) -> MithrilResult<()> {
+        let params: ConfigParameters = context.config_parameters()?.add_source(self)?;
+        let prepared_command: PreparedCardanoDbDownload = self.prepare(&params)?;
```

Comment:
You don't need to explicitly specify the types.
```suggestion
        let params = context.config_parameters()?.add_source(self)?;
        let prepared_command = self.prepare(&params)?;
```